studioclass-backend/src/utils/gCloud.js

```diff
@@ -1,15 +1,15 @@
 import { Storage } from '@google-cloud/storage';
 import { join } from 'path';
 
-// const gc = new Storage({
-//   credentials: JSON.parse(process.env.GCS_KEYFILE),
-//   projectId: 'fleet-petal-329519'
-// });
-
-const gc = new Storage({
-  keyFilename: join(__dirname, '../../fleet-petal-329519-cb1e7f8f25e8.json'),
-  projectId: 'fleet-petal-329519'
-});
+const gc = process.env.NODE_ENV === 'production'
+  ? new Storage({
+    credentials: JSON.parse(process.env.GCS_KEYFILE),
+    projectId: 'fleet-petal-329519'
+    })
+  : new Storage({
+    keyFilename: join(__dirname, '../../fleet-petal-329519-cb1e7f8f25e8.json'),
```

Comment:
Consider storing things like project IDs in environment variables, which can be changed agnostic to your source code.

Reply:
Adding all of these keys/secrets to env vars also alleviates the need for the ternary operator here. 

This also adds to dev/prod parity: [12 factor app](https://12factor.net/)

Reply:
Yeah, you're right it does make more sense to do that. The only problem I've run into with this is that the keyFile for setting up the storage object is JSON, and I haven't been able to figure out how to use JSON in a .env file. Any suggestions?

Reply:
Hey! Env vars are read into Node as a string, but Node provides ways to convert strings to JSON (namely stringify and parse). Store the json as a string, then build the object like so:

```javascript
const storageParams = {
  keyFilename: JSON.parse(process.env.keyFilename,
  ...
}

new Storage(storageParams)
```